packages/manager/modules/pci/src/projects/project/training/dashboard/dashboard.scss

```diff
@@ -0,0 +1,14 @@
+.ellipsis {
+  max-width: 100px;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+
+.big-number {
+  line-height: 1.25;
+  letter-spacing: 0;
+  font-weight: 400;
+  font-size: 36px;
+  color: #00185e;
+  margin: 0 0 16px;
```

Comment:
Could we use some ui-kit variables maybe

Reply:
how can I use it ?

Reply:
`@import @ovh-ux/ui-kit/dist/sass/_tokens.scss` I think

Reply:
https://github.com/ovh/manager/blob/master/packages/manager/modules/account-sidebar/src/index.scss#L11

Reply:
You can find an example here: https://github.com/ovh/manager/blob/master/packages/manager/modules/hub/src/variables.scss#L1

Reply:
fixed